test/truefi2/PoolFactory.test.ts

```diff
@@ -327,12 +327,12 @@ describe('PoolFactory', () => {
       const pool2 = await factory.pool(token2.address)
       const pool3 = await factory.pool(token3.address)
       const pool4 = await factory.pool(token4.address)
+
+      await expect(factory.isSupportedPool(pool3)).not
       await factory.supportPool(pool3)
+      await expect(factory.isSupportedPool(pool3))
```

Comment:
Are you sure this syntax works? Never seen it being used

Reply:
I don't think it works, after removing .not the test is still passing

Reply:
The correct way would be like this
expect(await factory.isSupportedPool(pool3)).to.be.false